src/components/ha-sidebar.ts

```diff
@@ -441,45 +505,45 @@ class HaSidebar extends LitElement {
           ${this.hass.localize("ui.notification_drawer.title")}
         </span>
         ${this.expanded && notificationCount > 0
-          ? html` <span class="notification-badge">${notificationCount}</span> `
+          ? html`
+              <span class="notification-badge" slot="meta"
+                >${notificationCount}</span
+              >
+            `
           : ""}
-      </paper-icon-item>
-    </div>`;
+      </ha-clickable-list-item>
+    `;
   }
 
   private _renderUserItem() {
-    return html`<a
-      class=${classMap({
-        profile: true,
-        // Mimick behavior that paper-listbox provides
-        "iron-selected": this.hass.panelUrl === "profile",
-      })}
-      href="/profile"
+    return html`<ha-clickable-list-item
+      class="profile"
+      .href=${"profile"}
```

Comment:
```suggestion
      href="profile"
```

Reply:
But `href` is a custom property I defined in `ha-clickable-list-item`. I think that should use a `.` and needs to be surrounded by `${}`